app/configuration/bundle_software_management.go

```diff
@@ -178,6 +178,8 @@ func (s Software) Execute(ctx context.Context, srv *Service, pkgManager software
 
 	// restart service if needed
 	if shouldRestart {
+
+		//TODO: We support systemctl and rc.scripts here (OpenWRT, RPM, DEB)
```

Comment:
What's the plan for resolving this?

Reply:
This doesn't need to be solved right now, but becomes important when (or if) we do the OpenWRT package manager integration. RHEL based systems use systemd.

Reply:
Should we then just drop the comment? If we need to track any future work, let's do it in Jira.